src/backend/vulkan/src/command.rs

```diff
@@ -248,25 +316,37 @@ impl command::RawCommandBuffer<Backend> for CommandBuffer {
 
     fn bind_graphics_pipeline(&mut self, pipeline: &n::GraphicsPipeline) {
         unsafe {
-            self.device.0.cmd_bind_pipeline(
-                self.raw,
-                vk::PipelineBindPoint::Graphics,
-                pipeline.0,
-            )
+            self.device
+                .0
+                .cmd_bind_pipeline(self.raw, vk::PipelineBindPoint::Graphics, pipeline.0)
         }
     }
 
-    fn bind_graphics_descriptor_sets(&mut self, layout: &n::PipelineLayout, first_set: usize, sets: &[&()]) {
-        unimplemented!()
+    fn bind_graphics_descriptor_sets(
+        &mut self,
+        layout: &n::PipelineLayout,
+        first_set: usize,
+        sets: &[&n::DescriptorSet],
+    ) {
+        let sets: SmallVec<[vk::DescriptorSet; 16]> = sets.iter().map(|set| set.raw).collect();
+
+        unsafe {
+            self.device.0.cmd_bind_descriptor_sets(
+                self.raw,                        // commandBuffer
+                vk::PipelineBindPoint::Graphics, // pipelineBindPoint
+                layout.raw,                      // layout
+                first_set as u32,                // firstSet
```

Comment:
similarly here, let's not have comments, and instead have local vars where needed:
`let dynamic_offsets = []; //TODO`